popup.css

```diff
@@ -0,0 +1,202 @@
+/* *, *::after, *::before {
+    box-sizing: border-box;
+} */
+
+.modal {
+  color: #344563;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  border: 1px solid black;
+  z-index: 10;
+  width: 315px;
+  max-height: 85vh;
+  min-width: 36%;
+  background-color: #fff;
+  transition: 200ms ease-in-out;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  justify-content: center;
+  overflow-y: scroll;
+}
+
+.modal.active {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.modal-title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 7rem;
+  padding-bottom: 2rem;
+}
+
+.modal-title h2 {
+  font-size: 25px;
+  line-height: 44px;
+  font-weight: 700;
+}
+
+.modal-title .close-btn {
+  font-size: 1.6rem;
+  border: none;
+  outline: none;
+  background-color: transparent;
+  color: hsl(212, 16%, 48%);
+  cursor: pointer;
+  margin-left: 2rem;
+  opacity: 0.7;
+}
+
+.modal-text {
+  padding: 1rem 0;
+  font-size: 16px;
+}
```

Comment:
While your design for the mobile popup is commendable, there are a few issues that need to be addressed in order to align your project with the Figma template:
- [ ] Please increase the size of the title to match the template :+1:.
- [ ] There is an empty space above the title that does not conform to the Figma design :+1:.
- [ ] To achieve consistency with the Figma template, it is necessary for the buttons to be centered and take up the entire width, rather than being aligned to the left :+1:.
By resolving these issues, you will ensure that your design closely matches the Figma template and maintains a cohesive visual presentation. Kindly make the required adjustments accordingly.

**Your design**

![Screenshot from 2023-05-17 15-16-24](https://github.com/aradradev/abdulsportfolio/assets/105117832/0b2c6f59-5a01-47b2-9d55-cc888bdfeab9)

**Figma**

![Screenshot from 2023-05-17 15-17-28](https://github.com/aradradev/abdulsportfolio/assets/105117832/a1360b49-5e6d-40a0-98f8-26e673bb0635)